propagator/propagator.go

```diff
@@ -50,6 +54,32 @@ func (e errInvalidHeader) Error() string {
 	return fmt.Sprintf("invalid header %s", e.header)
 }
 
+// CloudTraceOneWayPropagator will extract Trace IDs from the
+// x-cloud-trace-context header, and inject the w3c-standard 'traceparent'
+// header.
+type CloudTraceOneWayPropagator struct {
+	propagation.TraceContext
```

Comment:
Do we need to embed TraceContext here?  If we use this the way I recommended in the readme above, we would end up calling `propagation.TraceContext.Inject()` twice.  Are there other uses-cases where we don't want to make a composite propagator that it is needed for?

Reply:
I left a longer response to this on your comment in the README. Lets continue the discussion there.